docs/src/main/asciidoc/logging.adoc

```diff
@@ -571,12 +564,13 @@ test {
 
 See also <<getting-started-testing.adoc#test-from-ide,Running `@QuarkusTest` from an IDE>>.
 
-[[logging-adapters]]
-== Logging adapters
+[[logging-apis]]
+== Using other logging APIs and adapters
```

Comment:
I guess this should then be "Use other logging APIs". I wouldn't put "adapter" in the heading because it doesn't really reflect the intent of the user; I'd keep it down below.

Reply:
+1